libs/miranum-core/src/lib/miranum-core.spec.ts

```diff
@@ -64,3 +64,36 @@ describe("deploy", () => {
             .catch(e => expect(e).not.toBeNull());
     });
 });
+
+describe("generate", () => {
+    interface FileHelper {
+        name: string;
+        type: string;
+        extension: string;
+        dir: string;
+    }
+
+    const filesToGenerate: FileHelper[] = [
+        {name: "my-process", type: "bpmn", extension: "bpmn", dir: ""},
+        {name: "my-decision-table", type: "dmn", extension: "dmn", dir: ""},
+        {name: "my-form", type: "form", extension: "form", dir: "forms"},
+        {name: "my-config", type: "config", extension: "config", dir: "configs"},
+        {name: "my-element-template", type: "element-template", extension: "json", dir: "element-templates"},
+    ];
+
+    for (const file of filesToGenerate) {
```

Comment:
Den for loop würde ich auch hier in die Tests mit reinnehmen, damit das von intellij automatisch richtig als run config erkannt wird.